src/Misc/Part.h

```diff
@@ -159,6 +159,8 @@ class Part
         bool Pefxbypass[NUM_PART_EFX]; //if the effects are bypassed
 
         int lastnote;
+        char loaded_file[256];
+        bool hasFilename;
```

Comment:
Is this variable necessary? Could you not just initialize `loaded_file` as an empty string and then check if `(*loaded_file == 0)`?

Reply:
good idea.